nopolinfo #0
location: FailureLocation{className='org.sonar.api.utils.command.CommandExecutorTest', failingMethods=[org.sonar.api.utils.command.CommandExecutorTest#stdErr_consumer_can_throw_exception, org.sonar.api.utils.command.CommandExecutorTest#stdOut_consumer_can_throw_exception], erroringMethods=[org.sonar.api.utils.command.CommandExecutorTest#should_use_working_directory_to_store_argument_and_environment_variable, org.sonar.api.utils.command.CommandExecutorTest#should_consume_StdOut_and_StdErr, org.sonar.api.utils.command.CommandExecutorTest#should_stop_after_timeout], failures=[FailureType{failureName='org.sonar.api.utils.command.CommandException', failureDetail='java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit1150748389560922161/should_consume_StdOut_and_StdErr"): error=13, Permission denied', isError=true}, FailureType{failureName='org.sonar.api.utils.command.CommandException', failureDetail='java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/forever.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit438052314335068142/should_stop_after_timeout"): error=13, Permission denied', isError=true}, FailureType{failureName='java.lang.AssertionError', failureDetail='
Expected: (an instance of org.sonar.api.utils.command.CommandException and exception with message a string containing "Error inside stdErr stream")
     but: exception with message a string containing "Error inside stdErr stream" message was "java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2276828455126742934/stdErr_consumer_can_throw_exception"): error=13, Permission denied"
Stacktrace was: org.sonar.api.utils.command.CommandException: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2276828455126742934/stdErr_consumer_can_throw_exception"): error=13, Permission denied
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:102)
 at org.sonar.api.utils.command.CommandExecutorTest.stdErr_consumer_can_throw_exception(CommandExecutorTest.java:97)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
 at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
 at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
 at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2276828455126742934/stdErr_consumer_can_throw_exception"): error=13, Permission denied
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:74)
 ... 13 more
Caused by: java.io.IOException: error=13, Permission denied
 at java.lang.UNIXProcess.forkAndExec(Native Method)
 at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
 at java.lang.ProcessImpl.start(ProcessImpl.java:134)
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
 ... 14 more
', isError=false}, FailureType{failureName='org.sonar.api.utils.command.CommandException', failureDetail='java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/echo.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2514086516581767915/should_use_working_directory_to_store_argument_and_environment_variable"): error=13, Permission denied', isError=true}, FailureType{failureName='java.lang.AssertionError', failureDetail='
Expected: (an instance of org.sonar.api.utils.command.CommandException and exception with message a string containing "Error inside stdOut stream")
     but: exception with message a string containing "Error inside stdOut stream" message was "java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2776052275225002500/stdOut_consumer_can_throw_exception"): error=13, Permission denied"
Stacktrace was: org.sonar.api.utils.command.CommandException: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2776052275225002500/stdOut_consumer_can_throw_exception"): error=13, Permission denied
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:102)
 at org.sonar.api.utils.command.CommandExecutorTest.stdOut_consumer_can_throw_exception(CommandExecutorTest.java:89)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
 at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
 at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
 at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2776052275225002500/stdOut_consumer_can_throw_exception"): error=13, Permission denied
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:74)
 ... 13 more
Caused by: java.io.IOException: error=13, Permission denied
 at java.lang.UNIXProcess.forkAndExec(Native Method)
 at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
 at java.lang.ProcessImpl.start(ProcessImpl.java:134)
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
 ... 14 more
', isError=false}], nbFailures=2, nbErrors=3}
status: NOPATCH
dateEnd: Sun Apr 16 07:44:47 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/main/java], projectClasspath='[Ljava.net.URL;@5b58ed3c', projectTests=[org.sonar.api.utils.command.CommandExecutorTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_ERRORING
----------



nopolinfo #1
location: FailureLocation{className='org.sonar.api.utils.command.CommandExecutorTest', failingMethods=[org.sonar.api.utils.command.CommandExecutorTest#stdErr_consumer_can_throw_exception, org.sonar.api.utils.command.CommandExecutorTest#stdOut_consumer_can_throw_exception], erroringMethods=[org.sonar.api.utils.command.CommandExecutorTest#should_use_working_directory_to_store_argument_and_environment_variable, org.sonar.api.utils.command.CommandExecutorTest#should_consume_StdOut_and_StdErr, org.sonar.api.utils.command.CommandExecutorTest#should_stop_after_timeout], failures=[FailureType{failureName='org.sonar.api.utils.command.CommandException', failureDetail='java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit1150748389560922161/should_consume_StdOut_and_StdErr"): error=13, Permission denied', isError=true}, FailureType{failureName='org.sonar.api.utils.command.CommandException', failureDetail='java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/forever.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit438052314335068142/should_stop_after_timeout"): error=13, Permission denied', isError=true}, FailureType{failureName='java.lang.AssertionError', failureDetail='
Expected: (an instance of org.sonar.api.utils.command.CommandException and exception with message a string containing "Error inside stdErr stream")
     but: exception with message a string containing "Error inside stdErr stream" message was "java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2276828455126742934/stdErr_consumer_can_throw_exception"): error=13, Permission denied"
Stacktrace was: org.sonar.api.utils.command.CommandException: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2276828455126742934/stdErr_consumer_can_throw_exception"): error=13, Permission denied
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:102)
 at org.sonar.api.utils.command.CommandExecutorTest.stdErr_consumer_can_throw_exception(CommandExecutorTest.java:97)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
 at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
 at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
 at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2276828455126742934/stdErr_consumer_can_throw_exception"): error=13, Permission denied
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:74)
 ... 13 more
Caused by: java.io.IOException: error=13, Permission denied
 at java.lang.UNIXProcess.forkAndExec(Native Method)
 at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
 at java.lang.ProcessImpl.start(ProcessImpl.java:134)
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
 ... 14 more
', isError=false}, FailureType{failureName='org.sonar.api.utils.command.CommandException', failureDetail='java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/echo.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2514086516581767915/should_use_working_directory_to_store_argument_and_environment_variable"): error=13, Permission denied', isError=true}, FailureType{failureName='java.lang.AssertionError', failureDetail='
Expected: (an instance of org.sonar.api.utils.command.CommandException and exception with message a string containing "Error inside stdOut stream")
     but: exception with message a string containing "Error inside stdOut stream" message was "java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2776052275225002500/stdOut_consumer_can_throw_exception"): error=13, Permission denied"
Stacktrace was: org.sonar.api.utils.command.CommandException: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2776052275225002500/stdOut_consumer_can_throw_exception"): error=13, Permission denied
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:102)
 at org.sonar.api.utils.command.CommandExecutorTest.stdOut_consumer_can_throw_exception(CommandExecutorTest.java:89)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
 at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
 at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
 at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
 at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/test/scripts/output.sh" (in directory "/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/target/junit2776052275225002500/stdOut_consumer_can_throw_exception"): error=13, Permission denied
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
 at org.sonar.api.utils.command.CommandExecutor.execute(CommandExecutor.java:74)
 ... 13 more
Caused by: java.io.IOException: error=13, Permission denied
 at java.lang.UNIXProcess.forkAndExec(Native Method)
 at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
 at java.lang.ProcessImpl.start(ProcessImpl.java:134)
 at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
 ... 14 more
', isError=false}], nbFailures=2, nbErrors=3}
status: NOPATCH
dateEnd: Sun Apr 16 07:44:52 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/SonarSource/sonarqube/220930911/sonar-plugin-api/src/main/java], projectClasspath='[Ljava.net.URL;@251c8145', projectTests=[org.sonar.api.utils.command.CommandExecutorTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: IGNORE_FAILING
----------



